iothub_client/src/iothubtransport_amqp_device.c

```diff
@@ -56,6 +57,10 @@ typedef struct DEVICE_INSTANCE_TAG
     size_t twin_msgr_state_change_timeout_secs;
     DEVICE_TWIN_UPDATE_RECEIVED_CALLBACK on_device_twin_update_received_callback;
     void* on_device_twin_update_received_context;
+
+    TICK_COUNTER_HANDLE tick_counter_handle;
+    tickcounter_ms_t last_stop_request_time;
+    size_t stop_delay_ms;
```

Comment:
```suggestion
    size_t stop_timeout_ms;
```

Reply:
This period of time is really a delay, not a timeout.